packages/plugin-e2e/CONTRIBUTING.md

```diff
@@ -137,6 +137,6 @@ yarn add @grafana/plugin-e2e@0.3.0-canary.623.aedff75.0
 yarn e2e:plugin
```

Comment:
you also want to use `npm run` here.

Reply:
This (step 5) happend in grafana/grafana where we use yarn.

Reply:
I know that it is written in the text above but just to make it super clear we could add a comment above yarn commands. Something like:

```bash
# In your local grafana development folder.
yarn add @grafana/plugin-e2e@0.3.0-canary.623.aedff75.0
yarn e2e:plugin 
```

Reply:
I would also add something to make it super clear that the developer should discard the changes being made after running those commands until we have shipped a stable version of the plugin-e2e package.